src/features/duplicates/components/ConfigureModal.tsx

```diff
@@ -96,9 +96,9 @@ const ConfigureModal: FC<ConfigureModalProps> = ({
             }}
           />
           <Box marginBottom={2} />
-          <Alert severity="info">
-            <AlertTitle>{messages.modal.infoTitle()}</AlertTitle>
-            {messages.modal.infoMessage()}
+          <Alert severity="warning">
```

Comment:
Maybe this warning should be hidden entirely when there are no conflicting values. In those cases, no data will be lost, and it can be confusing for the user that the message refers to "the values you discard" when there has been no UI in which any values could have been discarded.

Could you try implementing logic that only shows this alert when there have been conflicting values (presented in the UI as dropdowns) above?